includes/Modules/Analytics_4.php

```diff
@@ -1048,7 +1048,12 @@ protected function create_data_request( Data_Request $data ) {
 			case 'GET:accounts':
 				return $this->get_service( 'analyticsadmin' )->accounts->listAccounts();
 			case 'GET:account-summaries':
-				return $this->get_service( 'analyticsadmin' )->accountSummaries->listAccountSummaries( array( 'pageSize' => 200 ) );
+				return $this->get_service( 'analyticsadmin' )->accountSummaries->listAccountSummaries(
+					array(
+						'pageSize'  => 200,
```

Comment:
You can test this PR by _temporarily_ setting the `pageSize` to something small like `5` to verify multiple pages are requested (assuming you have a Google account with more than 5 GA accounts :)